examples/jsm/renderers/nodes/lights/LightNode.js

```diff
@@ -0,0 +1,82 @@
+import Node from '../core/Node.js';
+import Object3DNode from '../accessors/Object3DNode.js';
+import PositionNode from '../accessors/PositionNode.js';
+import ColorNode from '../inputs/ColorNode.js';
+import FloatNode from '../inputs/FloatNode.js';
+import OperatorNode from '../math/OperatorNode.js';
+import MathNode from '../math/MathNode.js';
+import { NodeUpdateType } from '../core/constants.js';
+import { punctualLightIntensityToIrradianceFactor } from '../functions/BSDFs.js';
+
+import { Color } from 'three';
+
+class LightNode extends Node {
```

Comment:
Would it make sense to make more concrete light node classes later? Meaning `LightNode` would be the base class and `PointLightNode`, `DirectionalLightNode` etc. derivations.

Reply:
Yes, it seems the way.